spring-batch-docs/asciidoc/retry.adoc

```diff
@@ -339,25 +343,25 @@ A `BackoffPolicy` is free to implement
     the backOff in any way it chooses. The policies provided by Spring Batch
     out of the box all use `Object.wait()`. A common use case is to
     backoff with an exponentially increasing wait period, to avoid two retries
-    getting into lock step and both failing - this is a lesson learned from
-    the ethernet. For this purpose Spring Batch provides the
+    getting into lock step and both failing (this is a lesson learned from
+    ethernet). For this purpose, Spring Batch provides the
     `ExponentialBackoffPolicy`.
 
 [[retryListeners]]
 
 
 === Listeners
 
-Often it is useful to be able to receive additional callbacks for
-    cross cutting concerns across a number of different retries. For this
-    purpose Spring Batch provides the `RetryListener`
-    interface. The `RetryTemplate` allows users to
-    register `RetryListeners`, and they will be given
-    callbacks with the `RetryContext` and
+Often, it is useful to be able to receive additional callbacks for
+    cross cutting concerns from a number of different retries. For this
```

Comment:
In this case I think `across` is used correctly.  Across is commonly used when discussing cross cutting.

Reply:
I was trying to avoid the redundancy of "cross...across". If you think our readers will better understand your version, then go for it.